sql/postgresql/init-policy-database.sql

```diff
@@ -1,7 +1,8 @@
 -- *********************************************************************
--- Initialize Policy Database Script
+-- Update Database Script
```

Comment:
I think we should change this back to "Initialize Policy Database Script", just so that there's no confusion